### DISCUSSION TITLE
Save game requests

### DISCUSSION THREAD

**Discussion by aknight2015** [bucanero/apollo-saves#128] [General]
Is there some place where we can request saves for specific games? I'm personally looking for a save for Darkest Dungeon.

**Comment by bucanero** [comment]
honestly, here you won't find many users filling requests, so I think your best option would be asking on some Discord servers that are focused on PS4 save games, or some online forums like playersquared.com or thetechgame.com

**Comment by aknight2015** [comment]
That helps a great deal. Thank you very much.
